servant-client/servant-client.cabal

```diff
@@ -28,6 +28,16 @@ library
     Servant.Client
     Servant.Common.BaseUrl
     Servant.Common.Req
+     -- fixme: shouldn't be part of the public API:
+    Servant.Client.PerformRequest.Base
+  other-modules:
```

Comment:
I'm generally in favour of exporting all modules, but naming them `Internal.Something`.